.github/workflows/test_release.yml

```diff
@@ -57,7 +68,7 @@ jobs:
           CODECOV_UPLOAD_TOKEN: ${{ secrets.AWS_ACCESS_KEY_ID }}
 
   release:
-    needs: build
+    needs: test
```

Comment:
can we have lint here as well? btw, the concurrent running of lint and test is super cool! Nice Work!

Reply:
added